ml-agents-envs/mlagents_envs/tests/test_envs.py

```diff
@@ -45,6 +45,7 @@ def test_port_defaults(
     )
     env = UnityEnvironment(file_name=file_name, worker_id=0, base_port=base_port)
     assert expected == env._port
+    env.close()
```

Comment:
These weren't an actual problem, but cause noisy errors about closed handles when the tests shut down.